vendor/fbtransform/transforms/xjs.js

```diff
@@ -11,6 +11,42 @@
 var Syntax = require('jstransform').Syntax;
 var utils = require('jstransform/src/utils');
 
+function commaAfterLastParen(value) {
+  var state = 'normal';
+  var commaPos = 0;
+  for (var i = 0; i < value.length; ++i) {
+    if (state === 'normal') {
+      if (value.charAt(i) === '/') {
```

Comment:
State-machine type of deal and all that... but this could be reduced to `value.substr(i, 2) === '//'`, etc. @zpao?